src/Sarif.Sarifer/AnalyzeSolutionCommand.cs

```diff
@@ -40,6 +39,10 @@ private void MenuCommandCallback(object caller, EventArgs args)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
+            // always cancel before start
+            this.cancellationTokenSource?.Cancel();
```

Comment:
>? [](start = 40, length = 1)

Do we need the null conditional operator? `cancellationTokenSource` was initialized in the ctor.

Reply:
I removed from the ctor, since we always have to start a new one each time.

---
In reply to: [533004150](https://github.com/microsoft/sarif-visualstudio-extension/pull/302#discussion_r533004150) [](ancestors = 533004150)

Reply:
Oh, ok, I see it now, thanks.

---
In reply to: [533004913](https://github.com/microsoft/sarif-visualstudio-extension/pull/302#discussion_r533004913) [](ancestors = 533004913,533004150)